src/scratch/ScratchSound.as

```diff
@@ -242,11 +243,7 @@ public class ScratchSound {
 			bitsPerSample = 4;
 		}
 		reduceSizeIfNeeded(1); // downsample or compress to reduce size before saving
-		if (soundID == WasEdited) {
-			// sound was edited; force md5 to be recomputed
-			__md5 = null;
-			soundID = -1;
-		}
+		soundID = -1;
```

Comment:
I made this change after comparing this code to similar code dealing with images. That code is approximately:
```js
if (imageID == WasEdited) { md5 = null; }
soundID = -1;
```
Clearing the MD5 isn't necessary here because it's automatically cleared at the same time the ID becomes `WasEdited`, so I just always set `soundID = -1;`.

Why is `soundID = -1` important? Copied from 86be65b284951f17b0a0e44adf238fd7b62adbfa's commit message:
> Additionally, the sound ID is now always set to -1 in `prepareToSave`. This ensures that the sound will be reallocated a slot in the list of saved sounds. Even if this sound has not changed another sound may have been added or removed, which could lead to this sound's old ID pointing to the wrong entry in the array.
>
> Note that the list of saved sounds is rebuilt each save, so there's no need to deallocate the entry corresponding to the sound's old ID. See `recordImagesAndSounds` in `ProjectIO.as` for more details.